analyzer/library/const.go

```diff
@@ -0,0 +1,5 @@
+package library
+
+const (
+	Jar = "jar"
```

Comment:
I moved the value under `types`.
https://github.com/aquasecurity/fanal/blob/support_config/types/const.go#L18

Reply:
ah no wonder, then I'll delete this and use the one from `types`